integration_testing/development_plugin/kolibri_plugin.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+
+from magicbus.plugins import SimplePlugin
+from magicbus.plugins.tasks import Autoreloader
+
+from kolibri.core.content.hooks import ShareFileHook
+from kolibri.core.device.hooks import CheckIsMeteredHook
+from kolibri.core.device.hooks import GetOSUserHook
+from kolibri.core.tasks.hooks import StorageHook
```

Comment:
If we're breaking things, may as well do them all at once! I think I had not made it more specific because it was in the tasks app, but doesn't hurt to be specific in the hook name too.

Reply:
I guess the other thing here is that the fact that it is used by the Storage class in the tasks app doesn't really matter to the consumer, what matters is that it gives you methods about CRUD events for jobs, so a name that better reflects that might be helpful.

Reply:
That's a great point!